extensions/wikia/ArticleVideo/scripts/jwplayer-autoplay-toggle.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.jwPlayerAutoplayToggle', ['wikia.featuredVideo.autoplay'], function (featuredVideoAutoplay) {
+
+	return function (playerInstance) {
+		var inAutoplayCountries = featuredVideoAutoplay.inAutoplayCountries,
+			// TODO extract it to separate module
+			autoplayCookieName = 'featuredVideoAutoplay';
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'autoplayCookieName' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "autoplayCookieName" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)